src/Enhavo/Bundle/UserBundle/EventListener/VerificationRequiredListenerSubscriber.php

```diff
@@ -31,6 +31,7 @@ public static function getSubscribedEvents(): array
     {
         return [
             UserEvent::POST_AUTH => 'onPostAuth',
+            UserEvent::PRE_AUTH => 'onPostAuth',
```

Comment:
Should be `pre` or `post` auth. You could use `pre`, but then the called function should als called `onPreAuth`